src/test/java/testsuite/regression/ConnectionRegressionTest.java

```diff
@@ -6818,6 +6818,13 @@ public int available() throws IOException {
             // In some older Linux kernels the underlying system call may return 1 when actually no bytes are available in a CLOSE_WAIT state socket, even if EOF
             // has been reached.
             int available = this.underlyingInputStream.available();
+            try {
+                if (Arrays.asList(SSLContext.getDefault().createSSLEngine().getEnabledProtocols()).contains("TLSv1.3")) {
```

Comment:
Do you know if there is some type of enumeration that we can leverage here instead of comparing to a String value? If not, would suggest we put this in a local method final variable.

Reply:
There are no enums available. Moved the check to the constructor.